app/src/main/java/com/ddoczi/tasky/agenda/presentation/detail/agenda/AgendaDetailScreen.kt

```diff
@@ -48,16 +49,15 @@ fun AgendaDetailScreen(
             onEvent(AgendaDetailEvent.OnClose)
         }
     }
-    //Saving
-//    LaunchedEffect(
-//        key1 = savedStateHandle.get("title"),
-//        key2 = savedStateHandle.get("description")
-//    ){
-//        onEvent(AgendaDetailEvent.OnUpdatedInformation(
-//            savedStateHandle.get("title") ?: "",
-//            savedStateHandle.get("description") ?: ""
-//        ))
-//    }
+    LaunchedEffect(
+        key1 = savedStateHandle.get("title"),
+        key2 = savedStateHandle.get("description")
```

Comment:
These keys aren't Compose state and will therefore be static keys for LaunchedEffect (therefore, this will likely just be called once). Instead, it would be better to observe them as Compose state:
```
val title by savedStateHandle.getStateFlow<String?>("title", null).collectAsStateWithLifecycle()
```